pkg/infra/kubernetes/manifests/deployment.yaml.tmpl

```diff
@@ -25,4 +25,7 @@ spec:
       - image: REPLACE_ME
         name: {{ .ExecUnitName }}
       serviceAccount: {{ .ServiceAccountName }}
-      serviceAccountName: {{ .ServiceAccountName }}
\ No newline at end of file
+      serviceAccountName: {{ .ServiceAccountName }}
+      nodeSelector:
+        network_placement: REPLACE_ME
+        eks.amazonaws.com/nodegroup: REPLACE_ME
```

Comment:
similarly here, i think its fine for network placement to be always put on for now, but this one should not exist on every manifest since we wont always have a node selector for node group